### ISSUE TITLE
Create troubleshooting documentation for installing the prototype kit

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/prototype-kit#45]
### Background

The Design System currently has a guidance page for the [ONS Prototype Kit](https://service-manual.ons.gov.uk/design-system/guidance/ons-prototype-kit). However, there is no guidance for troubleshooting common issues in installation and setup. For example, how to resolve build errors, console errors, conflicts and account issues. There is information available about how to contact the team if further support is needed. 

We did receive feedback in a recent survey regarding this:

> The prototype kit is great to use but the initial set up was painful and needed support to do so as the guidance on the Design System was not accurate 

### Tasks
1. Identify the main pain points experienced by users when setting up the prototype kit
2. Document guidance for solving the main issues experienced
3. Possible requirement for content design to review
4. Publish the additional guidance to the "Installing the prototype kit" section

### Acceptance Criteria
We have identified the main issues users experience when installing the prototype kit and have guidance available on the [ONS Prototype Kit](https://service-manual.ons.gov.uk/design-system/guidance/ons-prototype-kit) page to resolve the most common issues.

**Comment by daniellecorke** [comment]
On hold - this work is blocked by #41. When #41 is complete, we can discuss this ticket again, refine it and point it.

**Comment by MagdalenaLarge** [comment]
https://jira.ons.gov.uk/browse/ONSDESYS-173

**Comment by Olatunde-Alalade-ONS** [comment]
Please track further updates to this ticket via [ONSDESYS-173](https://officefornationalstatistics.atlassian.net/browse/ONSDESYS-173)

[ONSDESYS-173]: https://officefornationalstatistics.atlassian.net/browse/ONSDESYS-173?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
